usagov/usagov-2021
# USAGOV-2133 readme improvements

## Jira Task
Updates and Improvements to the README.md file


https://cm-jira.usa.gov/browse/USAGOV-2133

## Description


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
Confirm all links work, instructions are accurate, and language is grammatically and syntactically correct.
Ensure styles are consistent.  
### Change Requirements

N/A

### Validation Steps
N/A

## Security Review

N/A


## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
